rnd/autogpt_server/autogpt_server/server/rest_api.py

```diff
@@ -91,6 +93,11 @@ def run_service(self):
             endpoint=self.get_graph_blocks,
             methods=["GET"],
         )
+        api_router.add_api_route(
```

Comment:
why have we forsaken routers here?

Reply:
What's routers ?

Reply:
Routers allow you to group like endpoints into their own files rather than whatever this is